sdk/substrate/src/lib.rs

```diff
@@ -340,7 +308,7 @@ pub struct Rpc {
     /// Maximum allowed subscriptions per rpc connection
     #[builder(default)]
     #[serde(default, skip_serializing_if = "sdk_utils::is_default")]
-    pub max_subs_per_conn: Option<usize>,
+    pub max_subs_per_conn: Option<u32>,
```

Comment:
You decided to not align CLI with `subspace-node` and `subspace-farmer`?

Reply:
I was not talking about the type, but rather CLI option names and such. We do not have `--max-subs-per-conn` in `subspace-node` anymore.